### PR TITLE
splitting list server details response

### PR DESCRIPTION
**Pull request by manishtomar**
Each part displays subset of response body since that is the biggest part of the event.

### CODE REVIEW

**Comment by manishtomar** [comment]
jenkins please run integration tests

File: otter/cloud_client.py [hunk 1]
```diff
@@ -345,13 +347,14 @@ def _log_it(result):
         # So we can link it to any non-cloud_client logs
         request_id = resp.request.headers.getRawHeaders(
             'x-otter-request-id', [None])[0]
-
+        resp_body = (
+            json.dumps(response_body_filter(json_body), sort_keys=True)
```
Comment (radix) [inline otter/cloud_client.py]: do you know why we wanted json in the first place? why not just always log it structured?

Comment (manishtomar) [inline otter/cloud_client.py]: I am guessing due to ES mapping issues. This way it always remains string and doesn't confuse ES indexes if for some reason different kind of values are returned.

**Comment by radix** [comment]
Thanks :)

**Pull request merged**